packages/@aws-cdk/aws-stepfunctions-tasks/test/emrcontainers/start-job-run.test.ts

```diff
@@ -0,0 +1,1173 @@
+          {
+            Action: 'logs:DescribeLogStreams',
+            Effect: 'Allow',
+            Resource: {
+              'Fn::GetAtt': [
+                'EMRContainersStartJobRunMonitoringLogGroup882D450C',
+                'Arn',
+              ],
+            },
+          },
+          {
+            Action: 'logs:DescribeLogGroups',
+            Effect: 'Allow',
+            Resource: 'arn:aws:logs:*:*:*',
+          },
+        ],
+        Version: '2012-10-17',
+      },
+      PolicyName: 'EMRContainersStartJobRunJobExecutionRoleDefaultPolicyCDBDF13E',
+      Roles: [
+        {
+          Ref: 'EMRContainersStartJobRunJobExecutionRole40B8DD81',
+        },
+      ],
+    });
+
+    // THEN
+    expect(stack.resolve(task.toStateJson())).toMatchObject({
+      Parameters: {
+        ConfigurationOverrides: {
+          MonitoringConfiguration: {
+            CloudWatchMonitoringConfiguration: {
+              LogGroupName: {
+                Ref: 'EMRContainersStartJobRunMonitoringLogGroup882D450C',
+              },
+            },
+            PersistentAppUI: 'ENABLED',
+            S3MonitoringConfiguration: {
+              LogUri: {
+                'Fn::Join': [
+                  '',
+                  [
+                    's3://',
+                    {
+                      Ref: 'EMRContainersStartJobRunMonitoringBucket8BB3FC54',
+                    },
+                  ],
+                ],
+              },
+            },
+          },
+        },
+      },
+    });
+  });
+
+  test('provided from user', () => {
+    // WHEN
+    const logGroup = logs.LogGroup.fromLogGroupName(stack, 'Monitoring Group', 'provided log group');
+    const s3Bucket = s3.Bucket.fromBucketName(stack, 'Monitoring Bucket', 'provided bucket');;
+    const prefixName = 'prefix';
+
+    const task = new EmrContainersStartJobRun(stack, 'EMR Containers Start Job Run', {
+      virtualCluster: VirtualClusterInput.fromTaskInput(sfn.TaskInput.fromText(clusterId)),
+      releaseLabel: ReleaseLabel.EMR_6_2_0,
+      jobDriver: {
+        sparkSubmitJobDriver: {
+          entryPoint: sfn.TaskInput.fromText('local:///usr/lib/spark/examples/src/main/python/pi.py'),
+          sparkSubmitParameters: '--conf spark.executor.instances=2',
+        },
+      },
+      monitoring: {
+        logBucket: s3Bucket,
+        logGroup: logGroup,
+        logStreamNamePrefix: prefixName,
+      },
+    });
+
+    // THEN
+    expect(stack).toHaveResourceLike('AWS::IAM::Policy', {
+      PolicyDocument: {
+        Statement: [
+          {
+            Action: [
+              's3:GetObject*',
+              's3:GetBucket*',
+              's3:List*',
+              's3:DeleteObject*',
+              's3:PutObject*',
+              's3:Abort*',
+            ],
+            Effect: 'Allow',
+            Resource: [
+              {
+                'Fn::GetAtt': [
+                  'EMRContainersStartJobRunMonitoringBucket8BB3FC54',
+                  'Arn',
```

Comment:
there is something broken in your implementation because this should just be "providedbucket"; looks like you are still creating a new bucket even if one is passed in